README.md

```diff
@@ -28,6 +28,25 @@ $ dnf install python{2,3}-conu conu-doc
 
 Please visit [our documentation](http://conu.readthedocs.io/en/latest/installation.html) for more info on installation.
 
+## Docker container
+
+You can try conu also in the container, but you have to mount docker socket and use `--privileged` option:
+
+```
+docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock:z --privileged modularitycontainers/conu:0.2.0 python3
+```
+
+```python
+from conu import DockerBackend
+:
```

Comment:
Here could be a comment `# Your python code` or `# Here you can play with conu` or some simple  conu example, e.g the helper function `get_container_output` introduced in https://github.com/fedora-modularity/conu/pull/154